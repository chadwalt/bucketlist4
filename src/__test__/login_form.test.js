import React from "react"
import renderer from "react-test-renderer"
import LoginForm from "../components/login_form"

describe("Login Snapshot", () => {
	it("renders correctly", ()=> {
		const rendered = renderer.create(<LoginForm/>)
		expect(rendered.toJSON()).toMatchSnapshot()
	})

})