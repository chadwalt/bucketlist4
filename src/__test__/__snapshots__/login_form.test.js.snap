// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Snapshot renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-md-4 col-md-offset-4"
    >
      <div
        className="login-panel panel panel-default"
      >
        <div
          className="panel-heading"
        >
          <h3
            className="panel-title"
          >
            Please Sign In
          </h3>
        </div>
        <div
          className="panel-body"
        >
          <form
            role="form"
          >
            <fieldset>
              <div
                className="form-group"
              >
                <input
                  autoFocus={true}
                  className="form-control"
                  name="username"
                  onChange={undefined}
                  placeholder="Username"
                  type="text"
                />
              </div>
              <div
                className="form-group"
              >
                <input
                  className="form-control"
                  name="password"
                  onChange={undefined}
                  placeholder="Password"
                  type="password"
                />
              </div>
              <a
                className="btn btn-lg btn-success btn-block"
                href="index.html"
                onClick={undefined}
              >
                Login
              </a>
              <a
                className="btn btn-lg btn-success btn-block"
                href="index.html"
                onClick={undefined}
              >
                Create Account
              </a>
            </fieldset>
          </form>
        </div>
      </div>
    </div>
  </div>
</div>
`;
